doc/classes/Joint3D.xml

```diff
@@ -13,19 +13,19 @@
 		<method name="get_rid" qualifiers="const">
 			<return type="RID" />
 			<description>
-				Returns the joint's [RID].
+				Returns the joint's internal [RID] from the [PhysicsServer3D].
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="exclude_nodes_from_collision" type="bool" setter="set_exclude_nodes_from_collision" getter="get_exclude_nodes_from_collision" default="true">
-			If [code]true[/code], the two bodies of the nodes are not able to collide with each other.
+			If [code]true[/code], the two bodies bound together cannot collide with each other.
 		</member>
 		<member name="node_a" type="NodePath" setter="set_node_a" getter="get_node_a" default="NodePath(&quot;&quot;)">
-			The node attached to the first side (A) of the joint.
+			Path to the first node (A) attached to the joint. The node must inherit [PhysicsBody3D].
```

Comment:
One of the node paths (A or B) is allowed to be empty, which has the effect of attaching the other body to an invisible static body without collision shapes, at the point given by the joint's position. (For example, that way you can easily make a pendulum with a PinJoint3D.) That should be phrased somehow nicely, maybe without the technical detail of the invisible static body without collision shapes. This also affects the description at the top of the file.

Reply:
Talk about obscure behavior. I have yet to use joints in any of my projects and I would have never known about this, at least at a glance. You may check again.